src/actions/register.js

```diff
@@ -241,16 +252,10 @@ async function performRegistration({ service, params }) {
 
   // assign alias
   if (alias) {
-    await assignAlias.call(service, {
-      params: {
-        username,
-        alias,
-        internal: true,
-      },
-    });
+    await assignAlias.call(service, { params: { username, alias, internal: true } });
```

Comment:
maybe its better to use service.dispatch so that validation & other middleware is triggered during this actiion